style.css

```diff
@@ -0,0 +1,3 @@
+body {
+  background-color: powderblue;
+}
```

Comment:
CSS :D 

One of my friend's advised me to look into CSS Grid to make websites look nicely formatted. There's a really good guide here made by Mozilla if you're interested: https://mozilladevelopers.github.io/playground/css-grid.

Reply:
Thanks! I'm still new to CSS so I thought I would at least try to add some style, I'll definitely look at the link to improve.